hail/build.gradle

```diff
@@ -44,21 +44,21 @@ String hailShortVersion = MAJOR.toString() + "." + MINOR.toString()
 String hailPipVersion = hailShortVersion + "." + PATCH + System.getProperty("hail.pip-version-suffix", "")
 String[] deployedSparkVersions = new File("deployed-spark-versions.txt").readLines()
 
-String sparkVersion = System.getProperty("spark.version", "2.2.0")
+String sparkVersion = System.getProperty("spark.version", "2.4.0")
 String breezeVersion = System.getProperty("breeze.version")
 String py4jVersion = System.getProperty("py4j.version")
 
 if (!(sparkVersion ==~ /^2\..*/))
-    ant.fail('Hail does not support Spark version ' + sparkVersion + '. Hail team recommends version 2.2.0.')
+    ant.fail('Hail does not support Spark version ' + sparkVersion + '. Hail team recommends version 2.4.0.')
 
 String scalaVersion = '2.11.8'
 String scalaMajorVersion = '2.11'
 
-if (sparkVersion == '2.2.0') {
+if (sparkVersion == '2.4.0') {
     if (py4jVersion == null)
-      py4jVersion = '0.10.4'
+      py4jVersion = '0.10.7'
     if (breezeVersion == null)
-      breezeVersion = '0.13.1'
+      breezeVersion = '0.13.2'
```

Comment:
we should keep the settings for old spark versions. I once went through and verified all the breezeVersions: https://github.com/hail-is/hail/pull/5130/files#diff-3c5fa39d01cc793629790d9a91cf9162L57

Basically 2.2.0 is `0.13.1` and all the other 2.x versions are `0.13.2`

Reply:
omg I spent 15 minutes looking for that yesterday across zulip/forum/email but didn't check the Make PR